containers/ddev-webserver/Dockerfile

```diff
@@ -78,7 +88,12 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -qq install -o Dpkg::Options::="--for
     zip
 
 ADD ddev-webserver-base-files /
-RUN curl -sSL "https://github.com/mailhog/MailHog/releases/download/v${MAILHOG_VERSION}/MailHog_linux_amd64" -o /usr/local/bin/mailhog
+RUN if [[ $TARGETPLATFORM == "linux/arm64" ]]; then \
+        # Issue for official arm64 support: https://github.com/mailhog/MailHog/issues/307
+        curl -sSL "https://github.com/dennisameling/MailHog/releases/download/v${MAILHOG_VERSION}-ddev/MailHog_linux_arm64" -o /usr/local/bin/mailhog; \
```

Comment:
Same for MailHog, can we also fork that? Forgot about this one, sorry

Reply:
mailhog also forked for you, thanks!

Reply:
Done: https://github.com/drud/MailHog/releases/tag/v1.0.1-ddev

Reply:
You are absolutely astonishing.